concepticondata/conceptlists/Healey-1970-364.tsv

```diff
@@ -0,0 +1,365 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Healey-1970-364-1	1	above	1741	ABOVE
+Healey-1970-364-2	2	adam's apple	802	ADAM'S APPLE
+Healey-1970-364-3	3	anus	1935	ANUS
+Healey-1970-364-4	4	arm	2121	ARM OR HAND
+Healey-1970-364-5	5	armpit	1886	ARMPIT
+Healey-1970-364-6	6	arrow	977	ARROW
+Healey-1970-364-7	7	ash	646	ASH
+Healey-1970-364-8	8	ask for/about	1309	ASK (INQUIRE)
+Healey-1970-364-9	9	at	1461	AT
+Healey-1970-364-10	10	avenge		
+Healey-1970-364-11	11	axe, stone	677	AXE
+Healey-1970-364-12	12	back	1291	BACK
+Healey-1970-364-13	13	backbone	805	SPINE
+Healey-1970-364-14	14	bad	1292	BAD
+Healey-1970-364-15	15	bake	183	BAKE
+Healey-1970-364-16	16	banana	868	BANANA
+Healey-1970-364-17	17	bat, fruit	235	FLYING FOX
```

Comment:
I'd unmapped because a flying fox is a type of fruit bat? I'm not sure about this.

Reply:
this is likely to be a flying fox (https://en.wikipedia.org/wiki/Great_flying_fox) rather than a fruit bat (https://en.wikipedia.org/wiki/Bulmer%27s_fruit_bat), as the author seems to use these synonymously